relay-general/tests/fixtures/payloads/android_unity.json

```diff
@@ -0,0 +1,369 @@
+{
+  "event_id": "39c94eefab9447229d70df71c7d41993",
+  "project": 5439417,
```

Comment:
Please  remove the following fields from the payload until we get the final ones from mobile team. These fields would not be sent in from an actual SDK and originate in server-side processing:

- `project`
- `dist` where empty
- `message` where empty
- `datetime`
- `_meta`
- `_metrics`
- `errors`
- `fingerprint`
- `grouping_config`
- `hashes`
- `key_id`
- `location`
- `metadata`
- `received`
- `version`

Reply:
I didn't bother with these... all the payloads should be replaced with data coming from the mobile team, I just put some payload there to have the code complete and the tests working. 

Initially I was looking for representative samples for each platform but after I couldn't find an IOS message I just copied the same message for all remaining platforms... so we have more problems than just additional fields.

I would suggest not to spend time fixing the messages since they are anyway not what we want and we intend to replace them shortly with real messages from the mobile team.